SUSE/rmt
# Integrate proofread changes into our code base

## Description

Apply the changes from proofread session int our codebase.

card: https://trello.com/c/5s5msoEv/2680-translation-round-ask-for-translations

**How to test this PR:**

- Grab yourself the pot file for glue, from Padraigs email (RE: [1/3] Translation round for SCC and RMT: Proofreading)
- Checkout i18n branch (for the source pot file) and compare:

```
diff new/glue.pot locale/glue.pot
```
- Check that all changes are reflected in this PR

## Change Type

*Please select the correct option.*
